src/utils/TestRunnerUtils.ts

```diff
@@ -0,0 +1,34 @@
+import Utils from './Utils';
+
+class TestRunnerUtils {
+
+    private constructor() {
+    }
+
+    static getTestRunId(
+        environment: string,
+        repoURL: string,
+        commitHash: string,
+        testRunKey: string) {
+        const testRunIdSeed = environment + '_' + repoURL + '_' + commitHash + '_' + testRunKey;
+        return Utils.generareIdFrom(testRunIdSeed);
+    }
+
+    static getDefaultTestRunId(
+        environment: string,
+        repoURL: string,
+        commitHash: string) {
+
+            /** todo: generate more unique id from parente process ? */
+            const runId = process.ppid + '_';
```

Comment:
Can we use globally generated UUID here and return it? I mean generate a uuid, store it in a global/static variable and use it here. So we will have same UUID during the same test process. 

However, if Jest loads Thundra module multiple times for each test-case environment (so in this case, global/static variable is re-initialized multiple times for each test), we may use process env vars as global storage here. I mean
```js
    static getDefaultTestRunId(
        environment: string,
        repoURL: string,
        commitHash: string) {
            let defaultTestRunId = process.env["THUNDRA_AGENT_DEFAULT_TEST_RUN_ID"]
            if (!defaultTestRunId) {
                defaultTestRunId = uuid.v4();
                process.env["THUNDRA_AGENT_DEFAULT_TEST_RUN_ID"] = defaultTestRunId;
            }
            
            return TestRunnerUtils.getTestRunId(
                environment,
                repoURL,
                commitHash,
                defaultTestRunId,
            );
    }
```

Reply:
process env which are set inside a test-case environment are not visible each other. I mean 

`process.env["THUNDRA_AGENT_DEFAULT_TEST_RUN_ID"] `

this is not in global storage, so it is empty each time and it will be re-initialized multiple times. This usage causes to generate different testRundId for each test-suite.